crates/api/src/transport.rs

```diff
@@ -273,13 +376,6 @@ pub trait Transport: 'static + Send + Sync + std::fmt::Debug {
 
     /// Dump network stats.
     fn dump_network_stats(&self) -> BoxFut<'_, K2Result<ApiTransportStats>>;
-
-    /// Increment the count of blocked messages for the given peer URL.
-    fn incr_blocked_message_count(
```

Comment:
This method was not actually required on the `Transport` trait, I was not thinking that through properly in the other PR (#335)